Dockerfile.alpine

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.13-alpine
```

Comment:
there too many files in the project root. Please, move both of Dockerfile-s into a new `/build` directory.

Reply:
Moved the Dockerfiles to top-level build directory.